### ISSUE TITLE
Verify Kuwait GDP per capita

### ISSUE THREAD

**Issue by sritchie73** [jennybc/gapminder#9]
I came across this while playing around with the Software Carpentry lesson materials:

```
library(gapminder)
library(ggplot2)
ggplot(data=gapminder, aes(x=year, y=gdpPercap, by=country)) + 
  geom_line(aes(color=continent)) +
  geom_line(data=gapminder[gapminder$country == "Kuwait",], aes(color=country), size=2)
```

![](http://i.imgur.com/TLjLkB0.png)

I suspect something wonky has happened with the `gdpPercap` data for Kuwait. I cannot reproduce this using data from gapminder.org. The data doesn't match "Income per person (GDP/capita, PPP$ inflation-adjusted)", whose trend suggests that the gdpPercap data may have been flipped over the time points in the dataset.

**Comment by jennybc** [comment]
I'm pretty sure this faithfully reflects the Excel spreadsheet they were distributing in 2008/2009, which I have included in this repo:

https://github.com/jennybc/gapminder/blob/master/data-raw/xls/gapdata001-1.xlsx

(look at the Kuwait row and your figure seems to reflect the data for 1952, 1957, etc.)

I extracted and cleaned the data via this script:

https://github.com/jennybc/gapminder/blob/master/data-raw/03_extract-from-excel-gdpPercap.r

Now, that's not to say the data is correct!

I will revisit the cleaning as fall approaches and close this and a couple other issues. But I think this represents a faithful data extraction from the original Excel sheet in 2008/2009. I have never understood the connection between these Excel sheets and what they show on their website -- it's completely opaque. Too bad they don't explain this or have an API :joy:.

I'm ambivalent about changing this. I've had this stable dataset for a long time now and have built lots of teaching materials around it. I may be satisfied as long as I am doing what I claim to do: have data that reflects that distributed by Gapminder.org circa 2008. I will never claim this data should be used for its intrinsic value as they are many sources of more recent, more correct data.
